src/components/card/Card.tsx

```diff
@@ -1,10 +1,57 @@
-import { Col, Row } from "antd";
-import React = require("react");
+import { Col, Divider, Row } from "antd";
+import { Carousel } from "antd";
+import * as React from "react";
+import { Card, UserRole} from "../../state/profile";
+import {BioData} from "./BioData";
+import {PreferenceComponent} from "./PreferenceSec";
+import {RealStateComponent} from "./RealStateSec";
 
-export const Card = (props: any) => (
-  <Row>
-    <Col>
-      <h3>this is a card {props.name}</h3>
-    </Col>
-  </Row>
-);
+import "./Card.scss";
+
+export const CardComponent = (props: Card) => {
+  return (
+    <Row type="flex" justify="center">
```

Comment:
This JSX element is huge :\ 
Can we break it down into smaller chunks of logic for readability? :)

Reply:
F**k this huge component...!!! how many f***ing components are we gonna create its gonna be lots of f***ing files/functions with redundant f***ing headers and imports :@:@:@:@@@@@@ do you break html files down too! consider this a nice sweet html file and accept my review plsss 😿 look the sun shines every day and its summer and because of these f***ing reviews I have to sit late in the library watch the sun going down from the library window T_T